implementations/micrometer-registry-statsd/src/main/java/io/micrometer/statsd/StatsdMeterRegistry.java

```diff
@@ -506,7 +506,7 @@ public Builder clock(Clock clock) {
          *                            timers, and histograms in the proprietary format.
          * @return This builder.
          */
-        public Builder lineBuilder(Function<Meter.Id, StatsdLineBuilder> lineBuilderFunction) {
```

Comment:
This is a breaking API change. Is there a way you can add an overload to `lineBuilder` that uses the `BiFunction` but still preserves the `Function` variant as well, possibly in deprecated form?

Reply:
Fixed!